puppet-client.html

```diff
@@ -239,7 +232,8 @@
             }
         },
         onLocalChange: function(patches) {
-            this.propagateDomBind(patches);
+            // todo #26
+            this.propagateDomBind({patches: patches, results: patches.map(function() { return {};})});
```

Comment:
Clients of this information (especially ppc where it notifies Polymer) expect (rightly) certain fields in this object. So right now ppc passes `result.removed` (which in this case turns out to be undefined) to Polymer. But if we wanted to allow the whole result object to be undefined, then ppc would need to check first if result is not undefined. At which point... what would poor ppc do? throw exception? swallow it with happy smile and pretend that API hasn't been violated?